bindings/rust/bench/README.md

```diff
@@ -11,25 +11,21 @@ All benchmarks are run in an idealized environment, using only a single thread a
 ../generate.sh
 
 # set up bench crate
-scripts/generate-certs.sh
-scripts/install-aws-lc.sh
+./scripts/generate-certs.sh
```

Comment:
Fun fact: you actually don't need the current directory "." to run scripts if they're not located in your current directory. "scripts/generate-certs.sh" works fine.

Reply:
Ya, but it's always irritated me that it breaks my autocomplete :(